test/resources/inventory-linking-exports-schemas/common/inventoryLinkingCommonTypes.xsd

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need those schemas here?

Reply:
Pasted them to be easy to reach. Could think about automatically updating if ILE changes them.